test/utils/simple_config_one_worker.yaml

```diff
@@ -1,6 +1,10 @@
 num_workers: 1
```

Comment:
Would be good to have a test with num_workers: 0 too as that case would be quite a lot different from the cluster mode I think